cdap-docs/reference-manual/source/http-restful-api/metrics.rst

```diff
@@ -612,7 +696,6 @@ difference calculated between the start and end times:
 - ``(endTime - startTime) >= 610 seconds``, resolution will be 1 minute; 
 - otherwise, resolution will be 1 second.
```

Comment:
this interval has been updated, now if difference > 36000 the resolution is in hours and > 600 the resolution will be in minutes.